x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="trayer fork with multi monitor support and cleaned up codebase"
+HOMEPAGE="https://github.com/sargon/trayer-srg"
+SRC_URI="https://github.com/sargon/${PN}/archive/${P/-srg/}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="${RDEPEND}
```

Comment:
DEPEND is now blank apart from virtual/pkgconfig (which should be in BDEPEND).

Here, DEPEND should be RDEPEND (and defined after, or swap the variable names), and put BDEPEND last with BDEPEND="virtual/pkgconfig"

Reply:
Fixed